docs/hosts/cisco.md

```diff
@@ -25,7 +25,7 @@ file line that looks like this in our configuration:
         ...
         vrf: management
     
-B. WRLinux NXOS (5) hosts will also require a user to be set on the
+B. Cisco Nexus hosts will also require a user to be set on the
```

Comment:
This requirement 'B' is true for both NX-OS and IOS XR. Requirement A above only applies to NX-OS (IOS XR *can* use a VRF but doesn't have to)